modules/model/shared/src/main/scala/observe/model/enums/Resource.scala

```diff
@@ -44,24 +44,26 @@ sealed abstract class Instrument(tag: String, ordinal: Int, label: String)
 
 object Instrument {
 
-  case object F2    extends Instrument("F2", 11, "Flamingos2")
-  case object Ghost extends Instrument("Ghost", 12, "GHOST")
+//  case object F2    extends Instrument("F2", 11, "Flamingos2")
+//  case object Ghost extends Instrument("Ghost", 12, "GHOST")
   case object GmosS extends Instrument("GmosS", 13, "GMOS-S")
   case object GmosN extends Instrument("GmosN", 14, "GMOS-N")
-  case object Gnirs extends Instrument("Gnirs", 15, "GNIRS")
-  case object Gpi   extends Instrument("Gpi", 16, "GPI")
-  case object Gsaoi extends Instrument("Gsaoi", 17, "GSAOI")
-  case object Niri  extends Instrument("Niri", 18, "NIRI")
-  case object Nifs  extends Instrument("Nifs", 19, "NIFS")
+//  case object Gnirs extends Instrument("Gnirs", 15, "GNIRS")
+//  case object Gpi   extends Instrument("Gpi", 16, "GPI")
+//  case object Gsaoi extends Instrument("Gsaoi", 17, "GSAOI")
+//  case object Niri  extends Instrument("Niri", 18, "NIRI")
+//  case object Nifs  extends Instrument("Nifs", 19, "NIFS")
```

Comment:
I removed the unsupported instrument to remove the need to deal with them in match statements. They must be restored as support for each instrument is put back in.

Reply:
Why not use the lucuma core Instrument?